[2024-03-02T15:32:34.705+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T14:40:00+00:00 [queued]>
[2024-03-02T15:32:34.730+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T14:40:00+00:00 [queued]>
[2024-03-02T15:32:34.731+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-03-02T15:32:34.732+0000] {taskinstance.py:1363} INFO - Starting attempt 8 of 12
[2024-03-02T15:32:34.733+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-03-02T15:32:34.768+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): Scrapping> on 2024-03-02 14:40:00+00:00
[2024-03-02T15:32:34.781+0000] {standard_task_runner.py:55} INFO - Started process 219 to run task
[2024-03-02T15:32:34.785+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Batching_Data', 'Scrapping', 'scheduled__2024-03-02T14:40:00+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/Batching_Data.py', '--cfg-path', '/tmp/tmphlbtmiq7']
[2024-03-02T15:32:34.789+0000] {standard_task_runner.py:83} INFO - Job 185: Subtask Scrapping
[2024-03-02T15:32:34.927+0000] {task_command.py:376} INFO - Running <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T14:40:00+00:00 [running]> on host 4502ada2f6f5
[2024-03-02T15:32:35.155+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bob Sebastian
AIRFLOW_CTX_DAG_ID=Batching_Data
AIRFLOW_CTX_TASK_ID=Scrapping
AIRFLOW_CTX_EXECUTION_DATE=2024-03-02T14:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=8
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-02T14:40:00+00:00
[2024-03-02T15:32:51.818+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Batching_Data.py", line 74, in batching_data
    label_encoder = load(os.getenv("label_encoder"))
  File "/home/airflow/.local/lib/python3.7/site-packages/joblib/numpy_pickle.py", line 658, in load
    obj = _unpickle(fobj, filename, mmap_mode)
  File "/home/airflow/.local/lib/python3.7/site-packages/joblib/numpy_pickle.py", line 577, in _unpickle
    obj = unpickler.load()
  File "/usr/local/lib/python3.7/pickle.py", line 1088, in load
    dispatch[key[0]](self)
  File "/usr/local/lib/python3.7/pickle.py", line 1385, in load_stack_global
    self.append(self.find_class(module, name))
  File "/usr/local/lib/python3.7/pickle.py", line 1426, in find_class
    __import__(module, level=0)
ModuleNotFoundError: No module named 'sklearn'
[2024-03-02T15:32:51.847+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=Batching_Data, task_id=Scrapping, execution_date=20240302T144000, start_date=20240302T153234, end_date=20240302T153251
[2024-03-02T15:32:51.875+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 185 for task Scrapping (No module named 'sklearn'; 219)
[2024-03-02T15:32:51.925+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-03-02T15:32:51.972+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
